supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{ProductNumberLink}/html/configuring_and_managing_identity_management/managing-kerberos-ticket-policies_configuring-and-managing-idm#role-of-idm-kdc_managing-kerberos-ticket-policies[The role of the IdM KDC].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[ldap]]
+==== image:images/yes.png[yes] LDAP (noun)
+*Description*: The Lightweight Directory Access Protocol (LDAP) is an open, vendor-neutral,  application protocol for accessing and maintaining distributed directory information services over a network. Part of this specification is a directory information tree (DIT), which represents data in a hierarchical tree-like structure consisting of the Distinguished Names (DNs) of directory service entries. LDAP is a "lightweight" version of the Directory Access Protocol (DAP) described by the ISO X.500 standard for directory services in a network.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[lightweight-sub-ca]]
+==== image:images/yes.png[yes] Lightweight sub-CA (noun)
+*Description*: In IdM, a lightweight sub-CA is a certificate authority (CA) whose certificate is signed by an IdM root CA or one of the CAs that are subordinate to it. A lightweight sub-CA issues certificates only for a specific purpose, for example to secure a VPN or HTTP connection.
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{ProductNumberLink}/html-single/configuring_and_managing_identity_management/index#restricting-an-application-to-trust-only-a-subset-of-certificates_configuring-and-managing-idm[Restricting an application to trust only a subset of certificates].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate-authorities[certificate-authorities]
+
+[discrete]
+[[managed-nodes]]
+==== image:images/yes.png[yes] managed nodes (noun)
+*Description*: The network devices, servers, or both that you manage with Ansible. Managed nodes are also sometimes called “hosts”. Ansible is not installed on managed nodes.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[ostree]]
+==== image:images/yes.png[yes] OSTree (noun)
+*Description*: A tool used for managing Linux-based operating system versions. The OSTree tree view is similar to Git and is based on similar concepts.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+
+[discrete]
+[[password-policy]]
+==== image:images/yes.png[yes] Password policy (noun)
+*Description*: A password policy is a set of conditions that the passwords of a particular IdM user group must meet. The conditions can include the following parameters:
+
+* The length of the password
+* The number of character classes used
+* The maximum lifetime of a password.
```

Comment:
Can you remove the period here to be consistent?

Reply:
+1